packages/mc-scripts/package.json

```diff
@@ -108,6 +109,7 @@
     "thread-loader": "3.0.4",
     "url": "^0.11.0",
     "vite": "~4.5.3",
+    "vite-bundle-analyzer": "0.12.1",
```

Comment:
I'm not sure if the patch will be used here when someone installs the `mc-scripts` package.

Reply:
Fair question. Couldn't find documentation on it. I must say I assume it would be applied.

Reply:
I don't know either but it won't matter in this case: the change in the source code is about a Typescript type which is not generic in the `rollup` version we're using but the dependency code expects it to be.
So, the generated code is exactly the same (no typings go there).